### DISCUSSION TITLE
How to pass the sunshine.conf path to sunshinesvc?

### DISCUSSION THREAD

**Discussion by xrain0610** [LizardByte/community#414] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

I modify install-service.bat like this:
set SERVICE_BIN="%ROOT_DIR%\tools\sunshinesvc.exe C:\Sunshine\config\sunshine.conf"
but , it not work, sunshine already use %ROOT_DIR%\config\sunshine.conf

how to let the windows service run use target conf file?

### Host Operating System

Linux

### Operating System Version

win 11 22H2

### Architecture

64 bit

### Sunshine commit or version

0.22.2

### Package

Windows - portable

### GPU Type

AMD

### GPU Model

6650xt

### GPU Driver/Mesa Version

4

### Config

```Shell
global_prep_cmd = [{"do":"cmd /C $(CONFPATH)/nircmd.exe setdisplay %SUNSHINE_CLIENT_WIDTH% %SUNSHINE_CLIENT_HEIGHT% 32 %SUNSHINE_CLIENT_FPS%","undo":"$(CONFPATH)/nircmd.exe setdisplay $(LOCAL_RESOLUTION)","elevated":"false"}]
origin_web_ui_allowed = wan
back_button_timeout = -1
min_log_level = 2
fps = [30,60,120]
resolutions = [
    1280x720,
    1920x1080,
    2560x1440
]
gamepad = x360
channels = 1
address_family = both
wan_encryption_mode = 0
lan_encryption_mode = 0
file_apps = C:/Sunshine/config/apps.json
credentials_file = C:/Sunshine/credentials/sunshine_state.json
log_path = C:/Sunshine/sunshine.log
pkey = C:/Sunshine/credentials/cakey.pem
cert = C:/Sunshine/credentials/cacert.pem
file_state = C:/Sunshine/credentials/sunshine_state.json
```

### Relevant log output

```shell
none
```

**Comment by ReenigneArcher** [comment]
As far as I know, the service executable doesn't pass arguments to sunshine.

[a comment by an automated account is not shown]
